GA/HeaderFiles/DataRead.h

```diff
@@ -20,50 +20,45 @@ void DataRead(vector<vector<vector<float> > >& dna_input,
   // This function reads in data from various files
 
   // Establish variables
-  int dna_garbage_end = 9;
-  ifstream generationDNA;
-  generationDNA.open("generationDNA.csv");
-  int csv_file_size = dna_garbage_end + (population * sections);
+  int dna_garbage_end = 9;                      // number of header lines
+  int csv_file_size = population * sections;
+  ifstream generationDNA("generationDNA.csv");
   string csv_content[csv_file_size];
-  string str_to_dbl;
+  string str_to_dbl;                            // string to double
 
-  // This loop reads through the .csv file line by line
-  // If we're in data (past line 9), it reads in the line
+  string dumpster;                              // getting rid of headers
+  for (int i=0; i<dna_garbage_end; i++) getline(generationDNA, dumpster);
```

Comment:
```suggestion
  for (int i=0; i<dna_garbage_end; i++) {
      getline(generationDNA, dumpster);
  }
```
It's generally better to keep one expression per line, and to always include `{}` even when the body of the loop has a single expression.